src/script/api/script_company.cpp

```diff
@@ -204,8 +204,10 @@
 
 	if (loan == GetLoanAmount()) return true;
 
+	Money amount = abs(loan - GetLoanAmount());
+
```

Comment:
```suggestion
	assert(amount >> 62 == 0);

```

Reply:
No, that suggestion is totally wrong.

Reply:
what about
`assert(GB((int64)amount, 62, 2) == 0);`

Reply:
Checking highest 2 bits is useless.

Reply:
I am guessing he was trying to ensure the lower (not higher) bits were zero'd. This is pointless, as we already check that the modulo of the interval is zero, and the interval is validated to have its lower bits zero. And yes, you can make code too pedantic :D